Sources_v3/StreamChatUI/ChatChannel/ChatMessageContentView.swift

```diff
@@ -51,10 +52,14 @@ open class ChatMessageContentView<ExtraData: ExtraDataTypes>: View, UIConfigProv
         .init()
         .withoutAutoresizingMaskConstraints
 
+    public private(set) lazy var errorIndicator = UIButton(type: .custom)
+        .withoutAutoresizingMaskConstraints
+
```

Comment:
This should probably be a custom type instead of setting the image in `defaultAppearance`

Reply:
introduced one in 1f84cffdc3bf89c97e082e340c723b2483bc91e4